doc/api/esm.md

```diff
@@ -206,12 +223,16 @@ This hook is called only for modules that return `format: "dynamic"` from
 the `resolve` hook.
 
 ```js
-export async function dynamicInstantiate(url) {
+export default function(parent) {
   return {
-    exports: ['customExportName'],
-    execute: (exports) => {
-      // get and set functions provided for pre-allocated export names
-      exports.customExportName.set('value');
+    dynamicInstantiate(url) {
+      return {
+        exports: ['customExportName'],
+        execute: (exports) => {
+          // get and set functions provided for pre-allocated export names
+          exports.customExportName.set('value');
+        }
+      };
```

Comment:
Similarly, perhaps this can be filtered to demonstrate chaining as well?

Reply:
fixed